acs-integration-tests/src/test/java/com/ge/predix/integration/test/SubjectResourceFixture.java

```diff
@@ -18,7 +18,11 @@
 import com.ge.predix.acs.rest.BaseResource;
 import com.ge.predix.acs.rest.BaseSubject;
 
-public class SubjectResourceFixture {
+public final class SubjectResourceFixture {
+    
+    private SubjectResourceFixture() {
+        //not called
```

Comment:
best to throw an exception if construction is attempted by a method of this class:

throw new IllegalAccessError("Class is non-instantiable");